wireapp/wire-android-reloaded
# fix: resample & rotate user avatar when picked from gallery - AR-1806

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Avatars picked from gallery where not resampled and rotated to normal orientation

### Solutions

Make both taking a new avatar picture or picking from library uniform in the processing steps


### Testing

Manual

#### How to Test

- With your phone's camera app, take a picture
- Open Wire Android Reloaded
- Go to your profile
- Change your profile picture by choosing from the gallery the picture taken in the first step
- Observe the avatar has the correct orientation

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
